### ISSUE TITLE
display the gzipped minified size too

### ISSUE THREAD

**Issue by jdalton** [kangax/jscritic#2]
jscritic should display the gzipped minified size too.

**Comment by kangax** [comment]
It's easy to show this when running through node (e.g. `require('child_process').exec('gzip -c ... > ...')`) but what about on a client side? Are there any good JS gzip libs that work in a browser?

/cc @XhmikosR @mathiasbynens

**Comment by XhmikosR** [comment]
Maybe https://github.com/nodeca/pako?

**Comment by XhmikosR** [comment]
@kangax:  https://github.com/sindresorhus/gzip-size

**Comment by mathiasbynens** [comment]
@XhmikosR https://github.com/sindresorhus/gzip-size doesn’t run client-side either. https://github.com/nodeca/pako claims to, though.

**Comment by XhmikosR** [comment]
Either way, I'm just throwing out ideas.

**Comment by sindresorhus** [comment]
Yeah, gzip-size is just for node. Just use pako. It's what gzip-size uses for the sync method.
